apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
We don't usually see orange buttons for teachers on the teacher dashboard. I can't actually think of a case. The moving students dialog (and saving data for sections) set the precedent for blue Save button. And in this dialog the orange buttons for locking/unlocking are actually the buttons we want them to use more, rather than saving the exceptions. Although I agree it's odd to have 4 orange buttons. But would also be weird if we had only 1 out of the 4 to be orange and have it change on some kind of rotation as we track which one you clicked last and encourage you to click the next one in the sequence. I think this is best choice given our current constraints, and when we start our designer (really really soon) have them take a crack at this.